pure-storage/info.json

```diff
@@ -0,0 +1,775 @@
+          "description": "Specify the comma-separated resource IDs to delete the corresponding array tags. If there is not at least one resource that matches each id element, an error is returned. Note: This parameter is required if Resource Names is not specified."
+        },
+        {
+          "name": "resource_names",
+          "title": "Resource Names",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Specify the comma-separated resource names to delete the corresponding array tags. If there is not at least one resource that matches each name element, an error is returned. Note: This parameter is required if Resource IDs is not specified.",
+          "description": "(Optional) Specify the comma-separated resource names to delete the corresponding array tags. If there is not at least one resource that matches each name element, an error is returned. Note: This parameter is required if Resource IDs is not specified."
+        },
+        {
+          "name": "keys",
+          "title": "Keys",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Specify the comma-separated tag keys to delete the corresponding array tags.",
+          "description": "(Optional) comma-separated tag keys to delete the corresponding array tags."
+        },
+        {
+          "name": "namespaces",
+          "title": "Namespaces",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Specify comma-separated namespaces to delete the corresponding tags.",
+          "description": "(Optional) Specify comma-separated namespaces to delete the corresponding tags."
+        }
+      ],
+      "output_schema": {}
+    },
+    {
+      "operation": "create_or_update_array_tags",
+      "title": "Create or Update Array Tags",
+      "annotation": "create_or_update_array_tags",
+      "description": "Creates or Updates Array Tags Pure Storage based on input parameters specified.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "tags",
+          "title": "Tags",
+          "type": "json",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "value": "[\n  {\n    \"key\": \"example_key\",\n    \"value\": \"value123\"\n  }\n]",
+          "tooltip": "Specify list of tags to be upserted.",
+          "description": "Specify list of tags to be upserted. For example [\n  {\n    \"key\": \"example_key\",\n    \"value\": \"value123\"\n  }\n]"
+        },
+        {
+          "name": "resource_ids",
```

Comment:
can we keep a select here cause any one of the parameter(i.e ID or Name) is req.??

Reply:
It is clear that we have to provide at least on of them but it is not clear whether can we provide both parameters at 
 same time or not ?